Makefile

```diff
@@ -194,7 +194,7 @@ start/kind:
 	@echo
 	@echo '>>> You need to manually run the following command in your shell: >>>'
 	@echo
-	@echo export KUBECONFIG="$$(kind get kubeconfig-path --name=kuma)"
+	@echo "kubectl cluster-info --context kind-kuma"
```

Comment:
From what I understand it prints information about the cluster of context kind-kuma. Does it switch to this this cluster so the next commands with kubectl will be executed on it?